python/cuml/test/test_fil.py

```diff
@@ -439,3 +439,53 @@ def test_lightgbm(tmp_path):
     fil_proba = np.reshape(fil_proba, np.shape(gbm_proba))
 
     assert np.allclose(gbm_proba, fil_proba, 1e-3)
+
+
+@pytest.mark.xfail(has_treelite() is False,
+                   reason="need to build treelite from source")
+def test_fil_skl_without_tl():
+
+    # settings
+    classification = True  # change this to false to use regression
+
+    X, y = simulate_data(100, 20, 2,
+                         random_state=123,
+                         classification=classification)
+    # identify shape and indices
+    train_size = 0.80
+
+    X_train, X_validation, y_train, y_validation = train_test_split(
+        X, y, train_size=train_size, random_state=0)
+
+    init_kwargs = {
+        'n_estimators': 20,
+        'max_depth': 5,
+        'max_features': 0.3,
+        'n_jobs': -1,
+    }
+
+    skl_model = RandomForestClassifier(**init_kwargs)
+    skl_model.fit(X_train, y_train)
+
+    skl_preds = skl_model.predict(X_validation)
+    skl_preds_int = np.around(skl_preds)
+    skl_proba = skl_model.predict_proba(X_validation)
```

Comment:
Is all of this necessary to test that we throw an exception? Should try to make this a minimal test to be quick and clear

Reply:
removed this test and updated the condition for the tests using `load_from_sklearn` function